client/my-sites/checkout/calypso-shopping-cart-provider.tsx

```diff
@@ -79,5 +79,11 @@ export default function CalypsoShoppingCartProvider( {
 
 function CalypsoShoppingCartMessages() {
```

Comment:
Question: would it even be better if we moved this to a separate file, and then move the `AsyncLoad` to load this component instead? That would save us an additional `import { useShoppingCart } from '@automattic/shopping-cart';` at minimum.

Reply:
I like it. 👍